### PR TITLE
feat:Siteinspector Template

### PR DESCRIPTION
**Pull request by bravohotel91**
Created a siteinspector template in response to #262

### CODE REVIEW

File: templates/siteinspector.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/siteinspector.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/siteinspector.png</Icon>
+  <Overview>A tool for catching spelling errors, grammatical errors, broken links, and other errors on websites.</Overview>
+  <Config Name="Config" Target="/app/" Default="/mnt/user/appdata/siteinspector" Mode="rw" Description="Config Location" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/siteinspector</Config>
+  <Config Name="Web UI" Target="808" Default="808" Mode="" Description="Web UI Port" Type="Port" Display="always" Required="true" Mask="false">808</Config>
+  <Config Name="Secret Key Base" Target="SECRET_KEY_BASE" Default="" Mode="" Description="Secret Key Base" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Enable Logging" Target="RAILS_LOG_TO_STDOUT" Default="true" Mode="" Description="Enable log reporting" Type="Path" Display="always" Required="true" Mask="false">true</Config>
```
Comment (Roxedus) [inline templates/siteinspector.xml:22]: not a path

File: templates/siteinspector.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Database Connection" Target="DATABASE_URL" Default="postgresql://postgres:postgres@postgres:5432/siteinspector" Mode="" Description="Postgres Database Connection String" Type="Path" Display="always" Required="true" Mask="false">postgresql://postgres:postgres@postgres:5432/siteinspector</Config>
```
Comment (Roxedus) [inline templates/siteinspector.xml:23]: not a path

File: templates/siteinspector.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Redis Connection" Target="REDIS_URL" Default="redis://redis:6379/0" Mode="" Description="Redis Connection String" Type="Path" Display="always" Required="true" Mask="false">redis://redis:6379/0</Config>
```
Comment (Roxedus) [inline templates/siteinspector.xml:24]: not a path

File: templates/siteinspector.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Port for Web UI" Target="PORT" Default="808" Mode="" Description="Enable log reporting" Type="Path" Display="advanced" Required="true" Mask="false">808</Config>
```
Comment (Roxedus) [inline templates/siteinspector.xml:25]: not a path

File: templates/siteinspector.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Sidekiq Currency" Target="SIDEKIQ_CONCURRENCY" Default="10" Mode="" Description="" Type="Path" Display="advanced" Required="true" Mask="false">10</Config>
```
Comment (Roxedus) [inline templates/siteinspector.xml:26]: not a path

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/siteinspector.xml (hunk 1 above)
Comment (bravohotel91) [reply templates/siteinspector.xml:22]: updated type

File: templates/siteinspector.xml (hunk 2 above)
Comment (bravohotel91) [reply templates/siteinspector.xml:23]: updated type

File: templates/siteinspector.xml (hunk 3 above)
Comment (bravohotel91) [reply templates/siteinspector.xml:24]: updated type

File: templates/siteinspector.xml (hunk 4 above)
Comment (bravohotel91) [reply templates/siteinspector.xml:25]: updated type

File: templates/siteinspector.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  <Config Name="Secret Key Base" Target="SECRET_KEY_BASE" Default="" Mode="" Description="Secret Key Base" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Enable Logging" Target="RAILS_LOG_TO_STDOUT" Default="true" Mode="" Description="Enable log reporting" Type="Variable" Display="always" Required="true" Mask="false">true</Config>
+  <Config Name="Database Connection" Target="DATABASE_URL" Default="postgresql://postgres:postgres@postgres:5432/siteinspector" Mode="" Description="Postgres Database Connection String" Type="Variable" Display="always" Required="true" Mask="false">postgresql://postgres:postgres@postgres:5432/siteinspector</Config>
+  <Config Name="Redis Connection" Target="REDIS_URL" Default="redis://redis:6379/0" Mode="" Description="Redis Connection String" Type="Variable" Display="always" Required="true" Mask="false">redis://redis:6379/0</Config>
+  <Config Name="Port for Web UI" Target="PORT" Default="808" Mode="" Description="Enable log reporting" Type="Port" Display="advanced" Required="true" Mask="false">808</Config>
```
Comment (nwithan8) [inline templates/siteinspector.xml:25]: Is this a duplicate of "Web UI" above?

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/siteinspector.xml (hunk 6 above)
Comment (bravohotel91) [reply templates/siteinspector.xml:25]: I believe they are separate. The Web UI is the port to use and the Port for Web UI is a variable.

**Review by Roxedus — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**